src/edu/csus/ecs/pc2/ui/ReportPane.java

```diff
@@ -1,4 +1,3 @@
-// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
```

Comment:
why remove, verses updating the year?

Reply:
Why indeed?  Mistake.  Fixed. [Commit](https://github.com/pc2ccs/pc2v9/pull/909/commits/ac4635f11971b2ebf29eb6f764a50d615603b0f4)